clients/gtest/rocblas_gtest_main.cpp

```diff
@@ -81,7 +81,7 @@ class ConfigurableEventListener : public TestEventListener
 
     void OnTestPartResult(const TestPartResult& result) override
     {
-        if(strcmp(result.message(), LIMITED_MEMORY_STRING))
+        if(strcmp(result.message(), LIMITED_MEMORY_STRING_GTEST) == 0)
```

Comment:
My fault for not reporting this earlier. I saw it and thought it was strange, but failed to comment on it, due to personal issues and an urgent request to approve the PR.